const catchAsyncError

exports.processPayment=catchAsyncError()